### PR TITLE
added flash msg display when step remove

### PR DESCRIPTION
**Pull request by rockstar777**
Fixed#345

### CODE REVIEW

**Comment by rockstar777** [comment]
Please review

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -177,8 +180,16 @@ function DefaultHtmlStepUi(_sequencer, options) {
   }
 
   function onRemove(step) {
-    step.ui.remove();
+    var x = step.ui.querySelector('#msg');
+    x.className = "show";
+    setTimeout(
+      function(){
```
[the remark this answers is not in the document]
Comment (rockstar777) [reply examples/lib/defaultHtmlStepUi.js]: sure

**Comment by tech4GT** [comment]
@rockstar777 Actually this issue is reserved for gci. Could you please take up an issue that is not labelled with `gci-candidate`. Thanks for the pr!

File: examples/lib/defaultHtmlStepUi.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tech4GT) [reply examples/lib/defaultHtmlStepUi.js]: @jywarren Actually this is a reserved for gci issue!

**Pull request closed without merging**